src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/AutoInsert/AttributeSnippetFormatOnAutoInsertProvider.cs

```diff
@@ -2,16 +2,20 @@
 // Licensed under the MIT license. See License.txt in the project root for license information.
 
 using System;
+using System.Diagnostics.CodeAnalysis;
 using System.Linq;
 using Microsoft.AspNetCore.Razor.Language;
 using Microsoft.AspNetCore.Razor.Language.Legacy;
 using Microsoft.AspNetCore.Razor.Language.Syntax;
 using Microsoft.AspNetCore.Razor.LanguageServer.Extensions;
 using Microsoft.AspNetCore.Razor.LanguageServer.Formatting;
+using Microsoft.Extensions.Logging;
 using Microsoft.VisualStudio.Editor.Razor;
 using OmniSharp.Extensions.LanguageServer.Protocol.Models;
 using Range = OmniSharp.Extensions.LanguageServer.Protocol.Models.Range;
 
+#nullable enable
```

Comment:
Why put this under the using statements? Pretty sure above is the norm.. though its not too important because hopefully we can just remove them all soon enough :)

Reply:
I too am excited for having everything be nullable.